<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search_Password</class>
 <widget class="QDialog" name="Search_Password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>410</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>262</width>
    <height>410</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Auto Savior : Password Generator</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>241</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_length">
          <property name="text">
           <string>Length (</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pwd_length_num">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_">
          <property name="text">
           <string>)</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="horizontal_pwd_length_Slider">
        <property name="minimum">
         <number>8</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="checkBox_letters">
          <property name="text">
           <string>Letters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_uppercase_n_lowercase">
          <property name="text">
           <string>Use uppercase and lowercase letters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_numbers">
          <property name="text">
           <string>Numbers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_symbols">
          <property name="text">
           <string>Symbols</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_compex_nb">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_statusbar_color">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>390</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #55ffff;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_complexity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Complexity : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_show_pwd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Password generated</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #0000ff;
color: white;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>    Password Generator</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>170</y>
     <width>301</width>
     <height>42</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Generate">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Copy_the_pwd">
      <property name="text">
       <string>Copy the password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_color_green">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>40</y>
     <width>441</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #00ff00;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_percent">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="Status_bar_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>label_color_green</zorder>
  <zorder>verticalLayoutWidget_3</zorder>
  <zorder>label_compex_nb</zorder>
  <zorder>label_statusbar_color</zorder>
  <zorder>label_complexity</zorder>
  <zorder>label_show_pwd</zorder>
  <zorder>label_title</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>label_percent</zorder>
  <zorder>Status_bar_text</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontal_pwd_length_Slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>pwd_length_num</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
